data/benefits/javascript/auvergne-rhone-alpes-bourse-merite.yml

```
@@ -1,6 +1,7 @@
 label: bourse au mérite
 institution: region_auvergne_rhone_alpes
-description: La bourse au mérite permet de récompenser les lycéen·ne·s et les
+description:
+  La bourse au mérite permet de récompenser les lycéens ou lycéennes et les
```

Dans une discussion avec @alizeeeeeee et @Shamzic  on parlait de mettre plutôt un "et" car le "ou" est souvent perçu exclusif dans le language usuel : 

```suggestion
  La bourse au mérite permet de récompenser les lycéens et lycéennes et les
```